content/chainguard/migration/migrating-node.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Migrating to Node.js Chainguard Images"
+linktitle: "Node Migration"
+type: "article"
+description: "Guidance on how to migrate Node.js Dockerfile workloads to use Chainguard Images"
+date: 2024-05-03T15:56:52-07:00
+lastmod: 2024-05-03T15:56:52-07:00
+draft: false
+tags: ["Images", "Product", "Conceptual"]
+images: []
+weight: 700
```

Comment:
Not sure what order we want these in, but we probably want something explicit. I put mine as 705, the PHP one is 700. Maybe 703 or 710?